### ISSUE TITLE
GLib-GObject-WARNING

### ISSUE THREAD

**Issue by catwell** [ender672/ruby-vips#2]
I get the following warning when I require ruby-vips:

```
>> require 'vips'
(irb:3279): GLib-GObject-WARNING **: cannot retrieve class for invalid (unclassed) type `<invalid>'
=> true
```

Do you know where it comes from?

**Comment by ender672** [comment]
What OS are you running, and what is your ruby version? I don't see that warning here, but perhaps I can reproduce it on your platform.

**Comment by catwell** [comment]
I got the error at work so I will not be able to try anything until Monday. It was on OS X, Ruby 1.8.7, with a recent (< 1 week) install of VIPS from Homebrew.

I only do simple thumbnailing with the library but it works fine, except for that warning.

**Comment by ender672** [comment]
I installed Ruby 1.8.7 and was unable to reproduce this. Perhaps I have a different version of one of the brew libraries. Can you paste your output for:
$ brew list --versions

**Comment by catwell** [comment]
cmake 2.8.1
gettext 0.17
glib 2.24.1 2.24.2
icu4c 4.4.1
ilmbase 1.0.1
imagemagick 6.6.4-5
jasper 1.900.1
jpeg 8a 8b
libevent 1.4.13
libexif 0.6.19
libgcrypt 1.4.5 1.4.6
libgpg-error 1.7 1.8 1.9
liboil 0.3.17
libpng 1.2.44
libtasn1 2.7
libtiff 3.9.2 3.9.4
nspr 4.8.4
openexr 1.6.1
pango 1.28.1
pcre 8.02 8.10
pkg-config 0.25
readline 6.0 6.1
spidermonkey 1.8.5
sqlite 3.6.23.1
vips 7.20.7
wxmac 2.8.11

I removed only some completely irrelevant stuff from this list. I use homebrew in my home directory and some package are MacPorts, I am not root on this system. I have confirmed it works correctly on a Linux box.
